src/star.css

```diff
@@ -0,0 +1,41 @@
+.stars {
+  display: flex;
+  text-align: center;
+  justify-content: space-between;
+  padding: 16px 16px 0;
+}
+
+.stars__container {
+  display: flex;
+}
+
+.stars__container--1 .stars__container-star--active:nth-child(1),
+.stars__container--2 .stars__container-star--active:nth-child(-n+2),
+.stars__container--3 .stars__container-star--active:nth-child(-n+3),
+.stars__container--4 .stars__container-star--active:nth-child(-n+4),
+.stars__container--5 .stars__container-star--active:nth-child(n+1) {
```

Comment:
You should use a relative path for the background-image: url('./images/star-active.svg') instead of url('images/star-active.svg').

Reply:
```
.stars__container--1 .stars__container-star--active:nth-child(-n + 1),
.stars__container--2 .stars__container-star--active:nth-child(-n + 2),
.stars__container--3 .stars__container-star--active:nth-child(-n + 3),
.stars__container--4 .stars__container-star--active:nth-child(-n + 4),
.stars__container--5 .stars__container-star--active:nth-child(-n + 5) {
  background-image: url("./images/star-active.svg");
}
```